pkg/cmd/cli/cmd/set/volume.go

```diff
@@ -237,10 +237,6 @@ func (v *VolumeOptions) Validate(args []string) error {
 
 func (a *AddVolumeOptions) Validate(isAddOp bool) error {
 	if isAddOp {
-		if len(a.MountPath) == 0 {
```

Comment:
if we don't have the info to validate it here, move the validation to a place where we know if we're adding a new volume below